protos/validator.proto

```diff
@@ -129,7 +129,7 @@ message Message {
     // content.
     MessageType message_type = 1;
 
-    // The identifier used to coorelate response messages to their related
+    // The identifier used to correlate response messages to their related
```

Comment:
You might have branched off of master, so I'm not sure if this commit will cause some weirdness